bower_components/jquery/.bower.json

```diff
@@ -0,0 +1,38 @@
+{
```

Comment:
Okay!  I think I fixed it.  There was a slight hitch - when I typed "bower install", the shell returned "ENOENT No bower.json present".  I went on StackOverflow, and the suggestion was to type bower init first, in order to set up a .json file in the root directory.  I did, and I'm not sure if I set it up correctly.  "bower install" did work properly after I finished "bower init", though.